interfaces/builtin/broadcom_asic_control.go

```diff
@@ -38,12 +38,25 @@ const broadcomAsicControlConnectedPlugAppArmor = `
 /dev/linux-user-bde rw,
 /dev/linux-kernel-bde rw,
 /dev/linux-bcm-knet rw,
+
+# The device info "0000:00:1c.0" can be found under the sysfs
+# and it's the parent device of first matched device node "0000:01:00.0"
+/sys/devices/pci[0-9]*/0000:00:1c.0/config r,
+/sys/devices/pci[0-9]*/0000:00:1c.0/{,subsystem_}device r,
+/sys/devices/pci[0-9]*/0000:00:1c.0/{,subsystem_}vendor r,
+
+/sys/bus/pci/devices/0000:00:1c.0/** r,
+/run/udev/data/+pci:0000:00:1c.0 r,
```

Comment:
Ah, right. There was another common detail I didn't take into account.
Using broader glob rules is a good fit, at least for now.
I take a note here.
RE: /sys/bus/pci/devices rule
I'd expect we need the same change for opengl interface.

Thanks for your explanation!